### ISSUE TITLE
OnMessageAttachmentsChanged launch event not running for inline replies & drafts Windows Outlook

### ISSUE THREAD

**Issue by alexm-plutoflume** [OfficeDev/office-js#3524]
The OnMessageAttachmentsChanged launch event is not running for **inline replies and drafts** for Windows Outlook. It seems to launch properly for popped out replies, drafts, and new mail.

## Your Environment

Windows Outlook

Microsoft® Outlook® for Microsoft 365 MSO (Version 2306 Build 16.0.16529.20100)

Windows 11 Enterprise 64-bit (10.0, Build 22621)
 
## Expected behavior
When adding or removing attachments in inline replies or drafts, we would expect the OnMessageAttachmentsChanged to launch.

## Current behavior
When adding or removing attachments in inline replies or drafts, OnMessageAttachmentsChanged does **not** launch.

## Steps to reproduce

1. Start composing a reply to an email inline, or composing a draft.
2. Add an attachment to the email.
3. Notice that the OnMessageAttachmentsChanged does **not** get triggered.

## Link to live example(s)
Have a manifest that uses an OnMessageAttachmentsChanged launch event with the following code:

```
async function onMessageAttachmentsChangedHandler(event: Office.AddinCommands.Event) {
  console.log("Attachments changed handler triggered");
  event.completed();
}

Office.actions.associate("onMessageAttachmentsChangedHandler", onMessageAttachmentsChangedHandler);

```

You should see that this does not trigger for inline reply/draft attachment changes but does trigger if the mail is popped out. Verified by checking runtime logging.

Thank you for taking the time to report an issue. Our triage team will respond to you in less than 72 hours. Normally, response time is <10 hours Monday through Friday. We do not triage on weekends.

**Comment by alexm-plutoflume** [comment]
Hi @exextoc, any updates here?

**Comment by exextoc** [comment]
Thanks for reporting this issue. We are able to reproduce this in-house. It has been put on our backlog. We unfortunately have no timelines to share at this point.

Internal tracking id: Office: 8155091

**Comment by exextoc** [comment]
We have fixed the bug in build 16.0.16731.10000.
The availability of this build depends on the release channel of all impacted users.
